### ISSUE TITLE
[SfDataGrid] [Web] - Vertical grid lines not visible in browsers on Windows OS (Edge / Chrome)

### ISSUE THREAD

**Issue by Abhijit-Revamp** [syncfusion/flutter-widgets#315]
Vertical gridlines are not visible in some of the browsers on Windows OS
The same are visible on macOS Safari, macOS Chrome etc.

Attached is the screen shot.

This happens when you have more rows that can fit on one screen. 
Example: Say you have about 300 rows in the table and 15 columns.  **All columns are integer and right aligned (for simplification)**.

When you scroll up and down using the vertical scroll bar (must be visible always), then suddenly this issue is visible. Also some horizontal gridlines are thick in middle (one of the cell bottom seems thicker than others - marked in the attached image)

Important: This is in **RELEASE** and DEBUG builds of the web app.

![image](https://user-images.githubusercontent.com/44320018/125628418-8ef1feb3-4740-4a04-a3bc-39bbcd223a98.png)

**Comment by RenugaSadagoban98** [comment]
Hi @Abhijit-Revamp ,  
Greetings from Syncfusion. 

 We have ensured the simple sample with (20 columns and more than 300 rows) with freeze pane on the web in both release and debug mode. But we couldn't able to reproduce the issue in our end. We need more details about your reported issue to do further analysis to reproduce it. Could you please provide a simple sample with the following details, 
	
Flutter channel (Stable, beta, dev or Master) ?  	
Channel version?	
SfDataGrid package?.	
Windows OS ? 

It will be helpful for us to check on it and provide you the solution at the earliest.

Regards,
Renuga devi

**Comment by Abhijit-Revamp** [comment]
@RenugaSadagoban98 - Let me try to get you a sample project as it will be difficult to reproduce

The only way is, you need to use the VERTICAL SCROLLBAR to drag up and down (don't use trackpad)

Flutter Channel : `Flutter (Channel master, 2.4.0-5.0.pre.55, on macOS 11.4 20F71 darwin-x64, locale en-IN)`
SfDataGrid package: syncfusion_flutter_datagrid: ^19.2.47-beta
Windows OS: Win 10 (latest one also 1 previous)

**Comment by RenugaSadagoban98** [comment]
Hi @Abhijit-Revamp ,

Thanks for your update.

In SfDataGrid, each grid cell is warped inside the container. We only provide the information about the border line to the Container->decoration-> border to draw a line for each cell. 
So, we ensured the framework widget to create like grid 
1) Used (Column & Row) widgets to create as like grid. 
2) Used Grid widget in flutter, in both case we faced the same issue of cell corner border are thick. in initial loading as well after scrolling. So, we reported the issue to framework.

GitHub issue link: https://github.com/flutter/flutter/issues/86558


Regards,
Renuga devi S
